packages/control/bat_all.py

```diff
@@ -288,14 +288,15 @@ def set_power_limit_controllable(self):
     def get_power_limit(self):
         if self.data.config.bat_control_permitted is False:
             return
-        chargepoint_by_chagemodes = get_chargepoints_by_chargemodes(CONSIDERED_CHARGE_MODES_ADDITIONAL_CURRENT)
+        chargepoint_by_chargemodes = get_chargepoints_by_chargemodes(CONSIDERED_CHARGE_MODES_CHARGING)
         power_of_chargepoints_by_chagemodes = sum(
-            [cp.data.get.power for cp in chargepoint_by_chagemodes if cp.data.get.power is not None])
+            [cp.data.get.power for cp in chargepoint_by_chargemodes if cp.data.get.power is not None])
+        # Falls aktive Steuerung an und Fahrzeuge laden und kein Überschuss im System ist,
+        # dann Speichereistung begrenzen.
         if (self.data.config.power_limit_mode != BatPowerLimitMode.NO_LIMIT.value and
-            len(chargepoint_by_chagemodes) > 0 and
+            len(chargepoint_by_chargemodes) > 0 and
```

Comment:
Soll die Speicher-Entladung gesperrt werden, sobald ein LP in einem Lademodus ist, in dem geladen werden kann oder nur, wenn wirklich geladen wird `data.data.cp_all_data.data.get.power`?